packages/esm-patient-chart-app/src/clinical-views/components/encounter-tile/encounter-tile.component.tsx

```diff
@@ -0,0 +1,100 @@
+import { CodeSnippetSkeleton, Tile, Column, Layer } from '@carbon/react';
+import React, { useMemo } from 'react';
+import styles from './tile.scss';
+import { groupColumnsByEncounterType } from '../../utils/helpers';
+import { useLastEncounter } from '../../hooks/useLastEncounter';
+import { LazyCell } from '../../../lazy-cell/lazy-cell.component';
+import { useTranslation } from 'react-i18next';
+import { type FormattedCardColumn } from '../../utils/encounter-tile-config-builder';
+import { useLayoutType } from '@openmrs/esm-framework';
+
+export interface EncounterTileColumn {
+  key: string;
+  header: string;
+  encounterUuid: string;
+  getObsValue: (encounter: any) => string | Promise<string>;
+  getSummaryObsValue?: (encounter: any) => string | Promise<string>;
+  encounter?: any;
+  hasSummary?: Boolean;
+}
+export interface EncounterTileProps {
+  patientUuid: string;
+  columns: Array<EncounterTileColumn>;
+  headerTitle: string;
+}
+
+export interface EncounterValuesTileProps {
+  patientUuid: string;
+  column: any;
+}
+
+const EncounterTileInternal: React.FC<EncounterTileProps> = ({ patientUuid, columns, headerTitle }) => {
+  const columnsByEncounterType = useMemo(() => groupColumnsByEncounterType(columns), [columns]);
+  const isTablet = useLayoutType() === 'tablet';
+
+  return (
+    <Layer className={styles.layer}>
+      <Tile className={styles.tile}>
+        <div className={isTablet ? styles.tabletHeading : styles.desktopHeading}>
+          <h4 className={styles.title}>{headerTitle}</h4>
```

Comment:
Any string displayed to the user needs to be translatable.

Reply:
The strings from the config are translated already